docs/source/quickstart.rst

```diff
@@ -139,6 +150,15 @@ Finally, you can ask PEPit to solve the system for you and return the worst-case
 
     pepit_tau = problem.solve()
 
+.. warning::
+    Performance estimation problems consists in reformulating the problem as an optimization problem, convex in a Gram
+    matrix G, and in function values F. The dimension of G is directly related to the number of points at which
+    the gradients are evaluated, and the differentiability of the function.
```

Comment:
Performance estimation problems consist in reformulating the problem of finding a worst-case scenario as a semidefinite problem. The dimension of the corresponding SDP is directly related to the number of function and gradient evaluations in a given code.